### ISSUE TITLE
When REST API returns HTTP 401, the log message should tell the user to check and configure a valid token in the configuration file

### ISSUE THREAD

**Issue by nirzilbermansisense** [kbbgl/e2ewd#40]
When REST API returns HTTP 401, the log message should tell the user to check and configure a valid token in the configuration file

**Comment by kbbgl** [comment]
Was added in unreachable area `tests/SisenseRESTAPIClient.java`:

```
catch (IOException e){

                logger.error("Query failed for " +
                        elastiCubeName + " with code " +
                        responseCode + " , error: " +
                        e.getMessage());

                if (responseCode == 401){
                    logger.warn("Check that the token in the configuration file is valid");
                }
                setCallSuccessful(false);
```

**Comment by kbbgl** [comment]
Fixed in 3.2.1:

![image](https://user-images.githubusercontent.com/14372649/70040480-4f834e00-15c4-11ea-8131-c29bc1167cf9.png)
